### ISSUE TITLE
Non-null value 'height' was null at $.data.attributes.nextStep.config.components[0].attributes.height

### ISSUE THREAD

**Issue by talaikis** [persona-id/persona-android-sdk#50]
Upgrade rn-persona from `2.1.2` to `2.2.5` gives:

```
e.e.a.b0.c in x at line 5
com.withpersona.sdk2.inquiry.ui.network.UiComponent_RemoteImage_AttributesJsonAdapter in a at line 6
com.withpersona.sdk2.inquiry.ui.network.UiComponent_RemoteImage_AttributesJsonAdapter in fromJson at line 1
e.e.a.b0.b in fromJson at line 3
com.withpersona.sdk2.inquiry.ui.network.UiComponent_RemoteImageJsonAdapter in a at line 5
com.withpersona.sdk2.inquiry.ui.network.UiComponent_RemoteImageJsonAdapter in fromJson at line 1
e.e.a.b0.b in fromJson at line 3
e.e.a.a0.b$b in fromJson at line 6
e.e.a.b0.b in fromJson at line 3
e.e.a.e in a at line 4
e.e.a.e$b in fromJson at line 1
e.e.a.b0.b in fromJson at line 3
com.withpersona.sdk2.inquiry.internal.network.NextStep_Ui_ConfigJsonAdapter in a at line 7
com.withpersona.sdk2.inquiry.internal.network.NextStep_Ui_ConfigJsonAdapter in fromJson at line 1
e.e.a.b0.b in fromJson at line 3
com.withpersona.sdk2.inquiry.internal.network.NextStep_UiJsonAdapter in a at line 5
com.withpersona.sdk2.inquiry.internal.network.NextStep_UiJsonAdapter in fromJson at line 1
e.e.a.b0.b in fromJson at line 3
e.e.a.a0.b$b in fromJson at line 6
e.e.a.b0.b in fromJson at line 3
com.withpersona.sdk2.inquiry.internal.network.CheckInquiryResponse_AttributesJsonAdapter in a at line 7
com.withpersona.sdk2.inquiry.internal.network.CheckInquiryResponse_AttributesJsonAdapter in fromJson at line 1
e.e.a.b0.b in fromJson at line 3
com.withpersona.sdk2.inquiry.internal.network.CheckInquiryResponse_DataJsonAdapter in a at line 4
com.withpersona.sdk2.inquiry.internal.network.CheckInquiryResponse_DataJsonAdapter in fromJson at line 1
e.e.a.b0.b in fromJson at line 3
com.withpersona.sdk2.inquiry.internal.network.CheckInquiryResponseJsonAdapter in a at line 4
com.withpersona.sdk2.inquiry.internal.network.CheckInquiryResponseJsonAdapter in fromJson at line 1
e.e.a.b0.b in fromJson at line 3
m.z.a.c in b at line 5
m.z.a.c in a at line 1
m.n in e at line 7
m.n$a in b at line 1
j.k0.g.e$a in run at line 7
java.util.concurrent.ThreadPoolExecutor in runWorker at line 1167
java.util.concurrent.ThreadPoolExecutor$Worker in run at line 641
java.lang.Thread in run at line 923
```

**Comment by SalvatoreT** [comment]
Hey @talaikis, sorry about that! We fixed that issue with [Android SDK v2.2.13](https://docs.withpersona.com/docs/android-sdk-v2-changelog#v2213---2022-03-30) and have a fix open internal to update the React Native SDK dependency to incorporate the change. The new version of React Native SDK should be out by Monday (Pacific Time) or so.

**Comment by EricLWang** [comment]
@talaikis the [React Native SDK version](https://docs.withpersona.com/docs/react-native-sdk-v2-changelogs) to incorporate the fix that @SalvatoreT mentioned was just merged and will be ready shortly :)
